src/inmanta/data/__init__.py

```diff
@@ -5267,38 +5183,12 @@ def copy_for_partial_compile(self, new_version: int) -> "Resource":
     async def get_released_resource_details(
```

Comment:
I think your changes to this method are OK, but I think the method deserves a critical review if it ends up staying. Could you add a note / ticket for the final stages of the epic? We can still decide to postpone it then, but let's at least make sure that we consider it at that point.

Reply:
https://github.com/inmanta/inmanta-core/issues/9066